### DISCUSSION TITLE
Accessing annotation version/dates for a saved variant/analysis search

### DISCUSSION THREAD

**Discussion by jxchong** [broadinstitute/seqr#2172] [Q&A]
Is there a way for end users to see the versioning/date-updated for annotations used in a saved variant search? For example, if I filtered for known pathogenic variants and come back 6 months later, I would want to know when/what version of ClinVar that search was run on because ClinVar would have changed in the intervening time, so there might be new variants to review/dismiss. Similarly, I would like to know what version of CADD/Eigen/SpliceAI/etc is being displayed when I view variant search results.

**Comment by hanars** [comment]
This is not currently supported in seqr

**Reply by jxchong** [reply]
If a search is saved, and then you revisit that search, and in the meantime, ClinVar or some other annotations were updated, which results are shown? The results from filtering against the annotations currently stored in seqr or the versions used in the initial saved search?

**Reply by hanars** [reply]
A saved search saves the search parameters but not the search results, and every time you use a saved search seqr actually performs a new search on whichever families/ projects you specify. As such the results will always show the latest available annotations.

**Reply by jxchong** [reply]
Ah. So is there a way to identify when a new variant appears in the results that wasn't there previously? I'm thinking about the scenario where a family is not solved after the first round of analysis, then an analyst goes back a few months later. How do they tell which variants were already considered and ruled out? I know you can use the Excluded flag, but that would require manually flipping that flag for every single variant, which could be 20+ variants per family. I guess a bulk flag function could help but I don't think that exists?

**Reply by hanars** [reply]
We do not see any value to bulk excluding variants. Unless an analyst has gone through and evaluated a variant theres really no reason to exclude it in the future, and if someone has taken the time to dig into a variant it doesn't really add any time to click one button to tag it
